docs/guides/docker.md

```diff
@@ -255,16 +247,41 @@ PersistentKeepalive = 15
 ```
 
 Once you've done that run `docker compose up -d` once more and you're all ready to go.
-
-Note that if you use qBittorrent and it's web UI you can optionally bind all connections to the Wireguard network inteface. This can be done by
+==-
+Note that if you use qBittorrent and its web UI you can optionally bind all connections to the VPN network inteface. This can be done by
 going to settings and navigating to the advanced tab all the way on the right. You should immediately see the options to bind your connections to a
-network interface, and to optionally bind your connections to the VPN IP.
+network interface, and to optionally bind your connections to the VPN IP. If using Gluetun, this should show up as `tun0`.
+
+## Accessing your containers
+
+Once everything is up and running, you should be able to access the Web UIs of your various Docker containers. Do so by opening a web browser and navigating to `http://ip:port`. For accessing a container on the same machine that's hosting it, `http://localhost:port` or `https://localhost:port` can be used. The `ip` field refers to the local machine ip (not your network's public ip). The `port` field is the port your container is configured to broadcast on, which is defined in `compose.yaml`. For example, qBittorrent will be at `http://localhost:8123` based on the file provided in this guide.
```

Comment:
I wouldn't include HTTPS in this because all of the containers in the stack are running HTTP out of the box and are not accessible via HTTPS without extra setup. If you wanted to include that then you would need to include why TLS matters, how to enable it per container, how to pop ports in the cases of where HTTPS required a different port, etc. It's probably out of the scope unless we want to drastically increase the length and complexity of this guide.